doc/api/cli.md

```diff
@@ -76,6 +76,21 @@ $ node --completion-bash > node_bash_completion
 $ source node_bash_completion
 ```
 
+### `-u`, `--conditions=condition`
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+Enable experimental support for custom conditional exports resolution
```

Comment:
If it's experimental, should the flag be `--experimental-conditions`?

Reply:
I would strongly prefer if not because I want to use and rely on it in my apps. It's experimental because the modules implementation is experimental. If the modules implementation were stable I would want to mark this similarly as stable. Having things go through an intermediate phase is fine for explicit feature flags, but I don't see why that should be necessary for all modules features by default by any means.